arangod/RocksDBEngine/RocksDBGeoIndex.cpp

```diff
@@ -282,7 +282,9 @@ class RDBNearIterator final : public IndexIterator {
           };
           auto* physical = _collection->getPhysical();
           // geo index never needs to observe own writes
```

Comment:
Is this comment now obsolete?

Reply:
No, nothing has changed w.r.t. "readOwnWrites" behavior here.
But we added a new flag that counts the number of bytes read here, but it is orthogonal to "readOwnWrites".